crates/holochain/src/conductor/interface/websocket.rs

```diff
@@ -1229,7 +1229,7 @@ mod test {
                     }
                 }
             })
-            .last()
+            .next_back()
```

Comment:
That's one silly name for a method!

Reply:
It is a little but it's actually a really useful clippy warning here. Apparently this happens to be a double-ended iterator and calling `last` actually forces it to run through the entire iterator. This is like the `.next()` method but starting from the other end of the iterator, so we end up just accessing one element. 

I do agree that without knowing the type, the name looks super odd, but the lint is really helping :)